build.psm1

```diff
@@ -848,6 +848,15 @@ function Start-PSPester {
         [switch]$IncludeFailingTest
     )
 
+    if (-not (Get-Module -ListAvailable -Name $Pester -ErrorAction SilentlyContinue)) {
```

Comment:
@daxian-dbw I made the module check a little stricter here than simply checking for the path.